src/System.Memory/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Integer.Signed.cs

```diff
@@ -19,8 +19,8 @@ private static bool TryFormatInt64(long value, ulong mask, Span<byte> buffer, ou
         {
             if (format.IsDefault)
             {
-                // Officially, the default is "G" but "G without a precision is equivalent to "D" so that's why we're calling the "D" helper.
-                return TryFormatInt64D(value, format.Precision, buffer, out bytesWritten);
+                // Officially, the default is "G" but "G without a precision is equivalent to "D" and so that's why we're using "D" (eliminates an unnecessary HasPrecision check)
+                format = 'D';
```

Comment:
Overriding the entire struct and calling the constructor here increases the method size.

What is wrong with the following?
```C#
char formatChar = format.Symbol;
if (format.IsDefault)
{
   formatChar = 'D';
}
switch (formatChar)
{
   ...
```

Reply:
What's wrong is that you'll pass the wrong `precision` down to the subformatter.

Reply:
Ah yes, if format.IsDefault is true, the precision == 0. But we want NoPrecision, not 0. Got it.